pdc/apps/changeset/views.py

```diff
@@ -80,7 +80,9 @@ def list(self, request, *args, **kwargs):
                     {
                         {
                             "author": "xxx",
+                            "requested_on": "2015-02-03T05:50:49.387Z",
                             "committed_on": "2015-02-03T05:51:17.262Z",
+                            "duration": "27.875",
```

Comment:
I don't think so. At most the unit could be mentioned in documentation, but putting it in the response does not seem very useful.